packages/experiments/src/components/fluent/theme/FluentColors.ts

```diff
@@ -26,215 +38,38 @@ export const GrayColors: any = {
-  greenCyan10: '#c2f2e9',
-  cyan230: '#005e50',
-  cyan220: '#006666',
-  cyan210: '#005b70',
-  cyan200: '#008272',
-  cyan190: '#038387',
-  cyan180: '#006f94',
-  cyan170: '#4a5459',
-  cyan160: '#009ca4',
-  cyan150: '#00b294',
-  cyan140: '#0099bc',
-  cyan130: '#00b7c3',
-  cyan120: '#5a686c',
-  cyan110: '#69797e',
-  cyan100: '#00bcf2',
-  cyan90: '#30c6cc',
-  cyan80: '#859599',
-  cyan70: '#31d2f7',
-  cyan60: '#61d6d6',
-  cyan50: '#a0aeb2',
-  cyan40: '#69eaff',
-  cyan30: '#91e5df',
-  cyan20: '#bac8cc',
-  cyan10: '#99ecff',
-  cyanBlue100: '#003966',
-  cyanBlue90: '#394146',
-  cyanBlue80: '#004e8c',
-  cyanBlue70: '#0063b1',
-  cyanBlue60: '#0078d7',
-  cyanBlue50: '#3a96dd',
-  cyanBlue40: '#83beec',
-  cyanBlue30: '#7ba7ff',
-  cyanBlue20: '#abc9ed',
-  cyanBlue10: '#b3dbf2',
-  blue150: '#11255e',
-  blue140: '#242466',
-  blue130: '#00188f',
-  blue120: '#19318d',
-  blue110: '#0027b4',
-  blue100: '#0037da',
-  blue90: '#203dbd',
-  blue80: '#0046ff',
-  blue70: '#2849ec',
-  blue60: '#4f6bed',
-  blue50: '#3b78ff',
-  blue40: '#7c96f9',
-  blue30: '#b5b5e2',
-  blue20: '#bebee5',
-  blue10: '#c3c3f4',
-  blueMagenta250: '#401b6c',
-  blueMagenta240: '#4e257f',
-  blueMagenta230: '#373277',
-  blueMagenta220: '#49397a',
-  blueMagenta210: '#32318c',
-  blueMagenta200: '#5c2e91',
-  blueMagenta190: '#49409a',
-  blueMagenta180: '#5e4a9d',
-  blueMagenta170: '#413eb3',
-  blueMagenta160: '#744da9',
-  blueMagenta150: '#5a4ebc',
-  blueMagenta140: '#735bc1',
-  blueMagenta130: '#8764b8',
-  blueMagenta120: '#4f4bd9',
-  blueMagenta110: '#6b69d6',
-  blueMagenta100: '#7160e8',
-  blueMagenta90: '#886ce4',
-  blueMagenta80: '#8378de',
-  blueMagenta70: '#8e8cd8',
-  blueMagenta60: '#a992d4',
-  blueMagenta50: '#9c89e9',
-  blueMagenta40: '#9a95de',
-  blueMagenta30: '#a68dff',
-  blueMagenta20: '#b4a0ff',
-  blueMagenta10: '#cfc4f5',
-  magenta160: '#5c005c',
-  magenta150: '#460f54',
-  magenta140: '#5c126b',
-  magenta130: '#800074',
-  magenta120: '#721481',
-  magenta110: '#9a0089',
-  magenta100: '#881798',
-  magenta90: '#b4009e',
-  magenta80: '#c239b3',
-  magenta70: '#b146c2',
-  magenta60: '#cc76ca',
-  magenta50: '#c182d1',
-  magenta40: '#e476c1',
-  magenta30: '#de94e0',
-  magenta20: '#e8a3de',
-  magenta10: '#dea2ed',
-  magentaPink70: '#6b0036',
-  magentaPink60: '#77004d',
-  magentaPink50: '#9b0062',
-  magentaPink40: '#bf0077',
-  magentaPink30: '#e3008c',
-  magentaPink20: '#e43ba6',
-  magentaPink10: '#f965bf',
-  pink50: '#970044',
-  pink40: '#c30052',
-  pink30: '#ea005e',
-  pink20: '#ee3f86',
-  pink10: '#edbed3'
-};
-
-export const CommunicationColors: any = {
-  communication90: '#004578',
-  communication80: '#005a9e',
-  communication70: '#106ebe',
-  communication60: '#0078d7',
-  communication50: '#2b88d8',
-  communication40: '#71afe5',
-  communication30: '#c7e0f4',
-  communication20: '#deecf9',
-  communication10: '#eff6fc'
+export const SharedColors: any = {
+  sharedPinkRed10: '#750b1c',
```

Comment:
Do we need to have `shared` as a prefix to everything? Feels like it could be simplified to something like `SharedColors.pinkRed10` or something like that.

Reply:
In that case is it supposed to be `NeutralColors.gray210` above?

Reply:
@oengusmacinog Sorry, where should it be gray210?